### PR TITLE
Fetch assets from contentful;

### PR DESCRIPTION
**Pull request by nadezhda**

### CODE REVIEW

File: src/content/index.ts [hunk 1]
```diff
@@ -20,4 +26,12 @@ export namespace Content {
   export function getSpace(): Promise<Space> {
     return getClient().getSpace();
   }
+
+  export function getAsset(id: string, query: any): Promise<Asset> {
```
Comment (sallar) [inline src/content/index.ts]: Remove the `query` since it's not used

File: src/content/index.ts [hunk 2]
```diff
@@ -20,4 +26,12 @@ export namespace Content {
...
+    return getClient().getAsset(id);
+  }
+
+  export function getAssets(query: any): Promise<ContentfulCollection<Asset>> {
```
Comment (sallar) [inline src/content/index.ts]: same as here

File: src/content/index.ts (hunk 1 above)
Comment (nadezhda) [reply src/content/index.ts]: Done

File: src/content/index.ts (hunk 2 above)
Comment (nadezhda) [reply src/content/index.ts]: Add query to getClient().getAssets(query) since I would assume we would need some assets collection by query, but not all

**Comment by sallar** [comment]
@nadezhda I need to release this

**Comment by sallar** [comment]
@nadezhda `@cxcloud/core@1.11.0` is now available

**Pull request merged**